server/data/data-jmap-cassandra/src/main/java/org/apache/james/jmap/cassandra/vacation/CassandraNotificationRegistryModule.java

```diff
@@ -45,6 +45,7 @@ public CassandraNotificationRegistryModule() {
                     .addPartitionKey(CassandraNotificationTable.ACCOUNT_ID, text())
                     .addClusteringColumn(CassandraNotificationTable.RECIPIENT_ID, text())
                     .withOptions()
+                    .comment("Stores TTLed registry of vacation notification being sent")
```

Comment:
You are right. It's our use of the table, but it is not enforced at the table level.